### PR TITLE
story/76: CycloneDX Gradle plugin for SBOM generation

### PR DESCRIPTION
**Pull request by ghost**
# CycloneDX Gradle plugin for SBOM generation

- added build.gradle plugin for CycloneDX sBOM generation
- [full plugin documentation for reference](https://github.com/CycloneDX/cyclonedx-gradle-plugin)

## NOTES for execution: 
- run `gradle cyclonedxBom` or `gradle cyclonedxBom -info` in the root directory. 
- **currently** does not excluded e2e, but it can.
- at current, files are generated to 'app/reports/sbom'.

## Issue 76
[CycloneDX Gradle Plugin](https://app.zenhub.com/workspaces/cdc-ti-633de70d939f6c00155f3722/issues/cdcgov/trusted-intermediary/76)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -11,6 +11,24 @@ plugins {
...
     id 'com.diffplug.spotless' version '6.11.0'
+    id 'org.cyclonedx.bom' version '1.7.2'
+}
+cyclonedxBom {
+
+    // Specified the type of project being built. Defaults to 'library'
+    projectType = "application"
+
+    // Boms destination directory (defaults to build/reports)
+    destination = file("reports/sbom")
```
Comment (halprin) [inline app/build.gradle:22]: I wouldn't overwrite this default.  I think it should go into the `build` directory with all of Gradle's other artifacts.  Then there isn't a need for the `README.md` either.

Comment (tjohnson7021) [reply app/build.gradle:22]: Sounds fine to me. I mainly created the Readme.md just to register the directory in github. I'll make those changes.

**Review by halprin — APPROVED** [review]
Sweet!

**Review by JohnNKing — APPROVED** [review]
Looks great to me!

**Pull request merged**